java/ql/src/semmle/code/java/frameworks/apache/Lang.qll

```diff
@@ -0,0 +1,11 @@
+/** Definitions related to the Apache Commons Lang library. */
+import semmle.code.java.Type
```

Comment:
Looks like there's an autoformat failure in this file:
```
ql/java/ql/src/semmle/code/java/frameworks/apache/Lang.qll is not correctly formatted
```
I think it's just missing an empty line between the file-level qldoc and the first import, but autoformat will tell.
Also, while we're at this file, could you replace the import with `import java`? We prefer always starting with `import <lang>` as the very first import in each file and this will include `semmle.code.java.Type`

Reply:
Why import all of `java` when I only need the one import? That sounds like creating tighter coupling to more imports than this class truly needs.

`Exec.qll` also only imports `semmle.code.java.Type`

Reply:
I've fixed it the way you've requested, but I'd still like to better understand the rationale here

Reply:
The optimiser will remove anything that's imported but not used, so no worries there. However, even though the optimiser tries its best to be as deterministic as possible, there are certain choices it has to make that needs some sort of arbitrary tiebreaker, which in the end can come down to the order in which declarations are encountered when traversing imports. Mostly this doesn't matter as the generated code will tend to be equivalent, but it might cause cache misses that can force re-evaluation of large parts of the libraries. We have been bitten by this problem before, and a good rule-of-thumb to avoid it is to start all ql files with `import <lang>` as the first import.